subset/network/network_tests.py

```diff
@@ -0,0 +1,91 @@
+import subprocess
+import time
+import sys
+
+name_of_tests = ['network.min_send', 'network.application.min_send', 'dhcp.long', 'ntp.update']
```

Comment:
seems like you're conflating "application" and "bacnet" -- I don't think it's safe to assume that....

Reply:
francessco and I discussed that using bacnet packets for the applcation layer may be acceptable. We can easily include other packets of interest in the range.

Reply:
bacnet may be *an* application layer, but it's not *the* application layer. An application layer is a pretty generic concept -- but, that may be fine here, if it's considered an abstraction around the implementation (bacnet, in this case)  and that in the future other application layers may be used.